src/devices/RealtimeClock/Devices/Pcf8563/Pcf8563.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Rtc
+{
+    /// <summary>
+    /// Realtime Clock PCF8563
+    /// </summary>
+    public class Pcf8563 : RtcBase
+    {
+        /// <summary>
+        /// PCF8563 Default I2C Address
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x51;
+
+        private I2cDevice _i2cDevice;
+        private bool _disposedValue = false;
+
+        /// <summary>
+        /// Creates a new instance of the PCF8563
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device used for communication.</param>
+        public Pcf8563(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice;
+
+            // Set "Normal Mode"
+            Span<byte> ctrl1Config = stackalloc byte[] { (byte)Pcf8563Register.PCF_CTRL1_ADDR, 0x00 };
+            _i2cDevice.Write(ctrl1Config);
+        }
+
+        /// <summary>
+        /// Read Time from PCF8563
+        /// </summary>
+        /// <returns>PCF8563 Time</returns>
+        protected override DateTime ReadTime()
+        {
+            // Sec, Min, Hour, Date, Day, Month & Century, Year
+            Span<byte> readBuffer = stackalloc byte[7];
+
+            _i2cDevice.WriteByte((byte)Pcf8563Register.PCF_SEC_ADDR);
+            _i2cDevice.Read(readBuffer);
+
+            return new DateTime(readBuffer[5] >> 7 == 1 ? 2000 + NumberTool.Bcd2Dec(readBuffer[6]) : 1900 + NumberTool.Bcd2Dec(readBuffer[6]),
```

Comment:
nit: this seems equivalent to `1900 + (readBuffer[5] >> 7) * 100 + NumberTool.Bcd2Dec(readBuffer[6])`